### PR TITLE
Fix highlight.js highlighting with language specified code blocks

### LINKED ISSUE #102: Code markdown doesn't respect language tag
Markdown for code blocks with a language specified such as ```csharp doesn't highlight the code for the specified language, instead it is using the highlight.js language detection logic.

I believe this is due to the use of `highlightElement`: https://github.com/MyNihongo/MudBlazor.Markdown/blob/4363af22b338b076b3a22180ad0e1da17949a8e7/src/MudBlazor.Markdown/Resources/MudBlazor.Markdown.js#L7 

Which from the highlight.js docs doesn't take any options, instead the language should be specified in the class attribute of the DOM node (see https://highlightjs.readthedocs.io/en/latest/api.html#highlightelement)

### PR DESCRIPTION
**Pull request by Lukazoid**
Fixes #102 

Before this commit the following would both render identically with go formatting as it was using highlight.js autodetection:

<pre>
```go
func main() {
    fmt.Println("Hello, 世界")
}
```
```plaintext
func main() {
    fmt.Println("Hello, 世界")
}
```
</pre>

After this commit the latter will correctly render as plaintext.

### CODE REVIEW

**Review by MihailsKuzmins — CHANGES_REQUESTED** [review]

File: src/MudBlazor.Markdown/Components/MudCodeHighlight.razor.cs [hunk 1]
```diff
@@ -61,7 +61,7 @@ protected override void BuildRenderTree(RenderTreeBuilder builder)
 
 		builder.OpenElement(i++, "pre");
 		builder.OpenElement(i++, "code");
-
+		builder.AddAttribute(i++, "class", $"language-{Language}");
```
[the remark this answers is not in the document]
Comment (Lukazoid) [reply src/MudBlazor.Markdown/Components/MudCodeHighlight.razor.cs]: Added in e4b06926277ca7118d486df2016ef125d9eaa5d7

**Review by MihailsKuzmins — APPROVED** [review]

**Comment by MihailsKuzmins** [comment]
OK, the code looks good, but now there are three failing tests. Please run the tests locally and fix them!
Please make sure to add a unit test where `Language` is null or empty to cover the new functionality.
```cs
[Fact]
public void RenderCodeBlockWithoutLanguage()
{
	const string value =
@"```
public bool IsMudBlazorCool()
{
	return true;
}
```";

	const string expected = ".....";

	using var fixture = CreateFixture(value);
	fixture.MarkupMatches(expected);
}
```

**Review by MihailsKuzmins — CHANGES_REQUESTED** [review]
OK, the code looks good, but now there are three failing tests. Please run the tests locally and fix them!
Please make sure to add a unit test where `Language` is null or empty to cover the new functionality.
```cs
[Fact]
public void RenderCodeBlockWithoutLanguage()
{
	const string value =
@"```
public bool IsMudBlazorCool()
{
	return true;
}
```";

	const string expected = ".....";

	using var fixture = CreateFixture(value);
	fixture.MarkupMatches(expected);
}
```

**Review by MihailsKuzmins — APPROVED** [review]
